javacutil/src/org/checkerframework/javacutil/Resolver.java

```diff
@@ -264,8 +287,12 @@ private Object getField(Object receiver, String fieldName) throws NoSuchFieldExc
     }
 
     private Symbol wrapInvocation(Method method, Object... args) {
```

Comment:
The method name should be changed: this version wraps an invocation on resolve, which is different from the new `wrapInvocation`, which takes a target.